### DISCUSSION TITLE
Node js version 12.22

### DISCUSSION THREAD

**Discussion by bharatjangid20** [aws/aws-sdk-js-v3#4180] [Q&A]
Can someone help me with the version of aws-sdk which supports node js version 12.22 ?

**Comment by yenfryherrerafeliz** [comment]
Hi @bharatjangid20, thanks for opening this discussion. Are you using AWS SDK for JavaScript V3?, if so, the end of support for node 12 in the V3 version ended on October 31, 2022, announced [here](https://github.com/aws/aws-sdk-js-v3/issues/3641), so that any release before that date may support node 12; however, you need to be aware that you may encounter bugs that were fixed after that date and also new AWS features released after that date may not be available in those old versions.

Thanks!
